tests/e2e/stepDefinitions/createSpaceContext.js

```diff
@@ -0,0 +1,55 @@
+const { Given, When, Then } = require("@cucumber/cucumber");
+const { CreateSpacePage } = require("../pageObjects/CreateSpacePage.js");
+
+const assert = require("assert");
+const createSpacePage = new CreateSpacePage();
+
+Then("user should see options to create space", async function () {
+  assert.ok(
+    await createSpacePage.createSpaceExixt(),
+    `Expected to see create space button bud could not find it`
+  );
+});
+
+When(
+  "user tries to create a space with name {string}",
+  async function (spaceName) {
+    await createSpacePage.createSpace(spaceName);
+  }
+);
+
+Then("redirects to search page", async function () {
+  assert.equal(
+    page.url(),
+    createSpacePage.searchURL,
+    `Expected to get ${createSpacePage.searchURL} but got : ${page.url()}`
+  );
+});
+
+Then(
+  "userspace {string} should be visible on option menu",
+  async function (spaceName) {
+    await createSpacePage.goToOverviewPage();
+    await assert.ok(createSpacePage.getSpace(spaceName));
+  }
+);
+
+Given("user is on space overview page", async function () {
+  await createSpacePage.goToOverviewPage();
+  assert.equal(
+    page.url(),
+    createSpacePage.spaceOverviewURL,
+    `Expected to get ${
+      createSpacePage.spaceOverviewURL
+    } but got : ${page.url()}`
```

Comment:
`Expected to get redirected to ${createSpacePage.spaceOverviewURL} but got redirected to  : ${page.url()}`